Latest NTypeScript build broken

Similar to some previous error reports where a new endpoint is exported but its types aren't 

```
../src/services/signatureHelp.ts(175,15): error TS4033: Property 'kind' of exported interface has or is using private name 'ArgumentListKind'.
```

Build : https://travis-ci.org/TypeStrong/ntypescript/builds/136283530#L326 :rose:

--- a/src/services/signatureHelp.ts
+++ b/src/services/signatureHelp.ts
@@ -165,7 +165,7 @@ namespace ts.SignatureHelp {
 
     const emptyArray: any[] = [];
 
-    const enum ArgumentListKind {
+    export const enum ArgumentListKind {
         TypeArguments,
         CallArguments,
         TaggedTemplateArguments

